シード追加

atsushi-sakuraya opened the pull request:
シード作成してみました。

・データの整合性確認　→西川さん対応
・skill_masterマイグレーションがありません　→西川さん対応
・masterマイグレーション名が「maseter」　→桜谷対応

よろしくお願いいたします。

Commit:
シード追加

nishikawamakio commented on database/seeds/MagicsMaseterTableSeeder.php:
```diff
@@ -0,0 +1,47 @@
+<?php
```
すみません、流石にコメントだけだと伝わらないですね。
こちら、以下URLを元に作ってみてもらってもよろしいでしょうか
https://www.triplestar.org/browse_various_data/magic_list.cgi

nishikawamakio commented on database/seeds/MysterysMaseterTableSeeder.php:
```diff
@@ -0,0 +1,47 @@
+<?php
```
すみません、こちら、以下URLの内容を当てはめる形でお願いできますでしょうか。
https://www.triplestar.org/browse_various_data/super_art_list.cgi

nishikawamakio commented on database/seeds/ItemsMaseterTableSeeder.php:
```diff
@@ -0,0 +1,41 @@
+<?php
+
```
すみません、こちらは、以下URLの内容でお願いできますでしょうか。
https://www.triplestar.org/browse_various_data/open_item_list.cgi

nishikawamakio commented:
データの整合性、確認しました。
マスター系以外は問題ありませんでした。
流石ですね。

マスター系のシードについては、
コメントで投入してほしいデータの一覧ページをURLで記載しました。

また、以下シードはゲームその物の知識が必要そうだと思ったので、
私の方で行おうと思います。
database/seeds/ArmorsMaseterTableSeeder.php
database/seeds/WeaponsMaseterTableSeeder.php 
database/seeds/BasePointTableSeeder.php
database/seeds/ExperiencePointMaseterTableSeeder.php
database/seeds/ImpactRangesMaseterTableSeeder.php
database/seeds/RangeTypesMaseterTableSeeder.php

Commit:
参考ページをもとに修正しました。
お手数ですがご確認お願いいたします。

■気になる部分
・奥義マスターテーブルに「消費FS」カラムは不要ですか？
・奥義マスターテーブルの消費スターカラムの綴りは「ster」でよいですか？
　星であれば「star」かなと思ったり。
・magics_maseter.impact_timeや
　items_master.effect_timeの単位時間は何になりますか？
　参考ページを見ると「分」「時間」「ターン」などがあったので気になりました。
■備考
maseterまだ直せてません...。
上記のカラム確認もあるので、次回持ち越しさせてください。

Commit:
time_units（単位表）、magic_master（魔法マスタ）シード追加

atsushi-sakuraya commented:
シードを作成しました。
・単位表
・効果範囲（ImpactRangesMaseterとRangeTypesMaseterを統合）
・魔法マスタ
https://github.com/nishikawamakio/reservation/pull/1/commits/cbbe07bcf31fe508884f3b4a336250505699538a

次は
・奥義マスタ
・スキルマスタ

nishikawamakio commented:
確認しました。
問題なかったので、マージしました。
